core/perf_test/PerfTestGramSchmidt.cpp

```diff
@@ -175,12 +175,23 @@ static void GramSchmidt(benchmark::State& state) {
   }
 }
 
+// FIXME_SYCL SYCL+Cuda reports "an illegal memory access was encountered"
```

Comment:
How is that related to using an in-order queue?

Reply:
I don't quite know yet. It works if put the additional fence before `memcpy` in again but that is the only test that seems affected and I'm leaning towards modifying the test rather than putting the `fence` in which might hide other problems in the future.